mw-webapp/src/component/accordion/Accordion.module.scss

```diff
@@ -0,0 +1,8 @@
+@import "src/styles/variables";
+
+.accordionRoot {
+    border-radius: $accordion-radius;
+    margin: 1rem auto;
```

Comment:
why 1 rem

Reply:
for beauty)
removed it)

Reply:
I mean why relation and not absolute value?